train_simulation/Person.py

```diff
@@ -1,3 +1,48 @@
+# from others.simulation_skeleton import Person
+import json
+from datetime import datetime
+import random
+# from Railway import Station
+
 
 class Person:
-    pass
\ No newline at end of file
+    def __init__(self, start_station, destination, time):  # I think that start station wil be good for us to generate report- can deleted if you are not agree
+        self.start_station = start_station
+        self.destination = destination
+        self.status = 'ready'
+        self.time = time
+        self.end_time = None
+        self.travel_time = None
+
+    def start_ride(self):
+        self.status = 'on board'
+
+    def end_ride(self):
+        self.status = 'done'
+        self.end_time = datetime.now()
+        self.travel_time = self.ride_duration()
+        # TODO cal method to register ride in a report
+        self.delete_person()
+
+    def delete_person(self):
+        del self
+
+    def ride_duration(self):
+        return self.end_time - self.time
+
+    @staticmethod
+    def create_passengers(critical_stations, time, n_passengers):
```

Comment:
Not working, can't use this method, call me

Reply:
tested several times, Please try
`critical_stations = ['Valby','Allerød','Holte']
passengers= Person.create_passengers(critical_stations, datetime.now(), 100)`

work fine at my side

Reply:
can't use it in main class, fix imports